content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/_cq_dialog/.content.xml

```diff
@@ -57,15 +57,16 @@
                                                 name="./navigationRoot"
                                                 value="${not empty cqDesign.navigationRoot ? cqDesign.navigationRoot : ''}"
                                                 required="{Boolean}true"/>
-                                            <skipNavigationRoot
+                                            <rootLevel
                                                 jcr:primaryType="nt:unstructured"
-                                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                                fieldDescription="Exclude the navigation root in the resulting tree, include its descendants only."
-                                                text="Exclude navigation root"
-                                                name="./skipNavigationRoot"
-                                                value="{Boolean}true"
-                                                checked="${not empty cqDesign.skipNavigationRoot ? cqDesign.skipNavigationRoot : true}"
-                                                uncheckedValue="{Boolean}false"/>
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                                fieldDescription="The levels below navigation root that are to be excluded. To include navigation root enter 0."
+                                                fieldLabel="Exclude Root Levels"
+                                                max="100"
+                                                min="0"
+                                                name="./rootLevel"
+                                                step="1"
+                                                value="${not empty cqDesign.rootLevel ? cqDesign.rootLevel : 1}"/>
```

Comment:
Same comment as above: would be great to be able to take `skipNavigationRoot` into consideration, if already defined.